src/coreComponents/physicsSolvers/surfaceGeneration/SurfaceGenerator.cpp

```diff
@@ -641,6 +644,10 @@ int SurfaceGenerator::separationDriver( DomainPartition & domain,
 
   for( int color=0; color<numTileColors; ++color )
   {
+
+    // MpiWrapper::barrier();
```

Comment:
Instead of leaving this debug output commented out, can we similarly log-level guard it and/or require a command-line argument to enable it?

Reply:
This stuff needs to be deleted after the follow on PR. So sure we could guard it with logs...but we never want to see this info again after the ParMETIS partition PR.